src/firmware_new/boards/frankie_v1/freertos.c

```diff
@@ -35,9 +35,9 @@
 /* USER CODE END Includes */
 
 /* Private typedef -----------------------------------------------------------*/
-/* USER CODE BEGIN PTD */
+/* USER CODE BEGIN PDF */
```

Comment:
I am guessing `PTD` stands for `Private TypeDef` here. What does `PDF` stand for in this context?

Reply:
This was a formatting error due to fix_formatting.sh not working properly on Ubuntu 20.04. I have reverted this change.